modules/nw-scaling-ingress-controller.adoc

```diff
@@ -5,33 +5,35 @@
 [id="nw-ingress-controller-configuration_{context}"]
 = Scaling an Ingress Controller
 
-An Ingress Controller can be manually scaled up or down based on
-requirements such as the requirement to increase throughput. `oc`
-commands are used to scale the `IngressController` resource.
-The following procedure provides an example for scaling up the
-default `IngressController`.
+An Ingress Controller can be manually scaled up or down based on routing
+performance or availability requirements such as the requirement to increase
+throughput. `oc` commands are used to scale the `IngressController` resource.
+The following procedure provides an example for scaling up the default
+`IngressController`.
 
 .Procedure
-. View the current number of available replicas for the default `IngressController`.
+. View the current number of available replicas for the default `IngressController`:
 +
 ----
 $ oc get -n openshift-ingress-operator ingresscontrollers/default -o jsonpath='{$.status.availableReplicas}'
-
+2
 ----
 
 . Scale the default `IngressController` to the desired number of replicas using
 the `oc patch` command. The following example scales the default `IngressController`
-to 3 replicas.
+to 3 replicas:
 +
 ----
 $ oc patch -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"replicas": 3}}' --type=merge
+ingresscontroller.operator.openshift.io/default patched
 ----
 
-. Verify the default `IngressController` has been updated with the desired
-number of replicas.
+. Verify the default `IngressController` has been updated with the expected
```

Comment:
s/Verify the/Verify that the
I think "expected" works. If you're interested in other opinions, the IBM Style guide suggests using "required" when you pull "desired," and I'd probably say "Verify that the default `IngressController" scaled to the number of replicas that you specified:"
> :no_entry_sign: desired adj
Do not use. Use “required” or “the <item> that you want.”

Reply:
I like that suggestion, thanks!